### ISSUE TITLE
Keyboard selection

### ISSUE THREAD

**Issue by monmarty** [PhilippC/keepass2android#1723]
I'm using Microsoft SwiftKey keyboard. If I want to edit any item then Keepass2Android offline force me to select keyboard. If I select Samsung keyboard or Microsoft SwiftKey it forces me again and again. Only possibility is to select Keepass2Android offline keyboard. My cell phone is Samsung Galaxy M21.

**Comment by PhilippC** [comment]
sounds like you have enabled settings - app - security - use built-in keyboard in KP2A?

**Comment by monmarty** [comment]
No, I want to use my favorite Microsoft Swiftkey but it's impossible to
select it - Application forces me to choose KP2A (If I choose any other
keyboars than KP2A it will go back to select again and again...)   . Any
idea how to solve this problem? Thanks.

po 20. 9. 2021 v 11:26 odesílatel PhilippC ***@***.***>
napsal:

**Comment by PhilippC** [comment]
can you explain when exactly this happens? BTW did you have a look at https://github.com/PhilippC/keepass2android/blob/master/docs/Documentation.md?
